code/modules/reagents/chemistry/reagents/drinks.dm

```diff
@@ -480,3 +480,27 @@
 	description = "Beloved of children and teetotalers."
 	color = "#E6CDFF"
 	taste_message = "grape soda"
+
+/datum/reagent/consumable/laughter
+	name = "Laughter"
+	id = "laughter"
+	description = "Some say that this is the best medicine, but recent studies have proven that to be untrue."
+	metabolization_rate = INFINITY
+	color = "#FF4DD2"
+
+/datum/reagent/consumable/laughter/on_mob_life(mob/living/carbon/M)
+	if(!iscarbon(M))
+		return
+	if(!M.silent)//cant laugh if you're mute
+		M.emote("laugh")
+		var/laughnum = rand(1,2)
```

Comment:
this whole bit would be better using`pick()` from a list instead...

In psuedocode... 
```
if female
   pick (c sound or d sound) to play
else
   pick (a sound or b sound) to play```

That'd make it look a lot cleaner imo.